### PR TITLE
6 - Add a comment list to each article view

### PR DESCRIPTION
**Pull request by Tiny-B**
This pr adds a list of comments to the full article view pages at the bottom of the page

### CODE REVIEW

File: src/components/article-components/CommentCard.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const CommentCard = ({ comment }) => {
+	return (
+		<article className='comment'>
+			<h4>{comment.author}</h4>
+			<h6>
+				<span>
+					{new Date(comment.created_at)
```
Comment (mallionaire) [inline src/components/article-components/CommentCard.jsx:7]: Nice formatting, there are several packages that can manage this sort of thing for us, you might want to take a look at moment.js or day.js if you'd like to refactor this.

File: src/components/article-components/ArticleComments.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+import { api } from '../../api';
+import CommentCard from './CommentCard';
+
+const ArticleComments = ({ articleId }) => {
+	const [comments, setComments] = useState([]);
```
Comment (mallionaire) [inline src/components/article-components/ArticleComments.jsx:6]: You might want to add some separate indication that comments are loading for your user, as the loading state in your Article component is not a guarantee that the api request for comments will have completed. But be careful to think about scenarios where an article doesn't have any comments 🤔

File: src/components/article-components/ArticleComments.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+						cols='30'
+						rows='4'
+					></textarea>
+					<button>Submit</button>
+				</form>
+			</div>
+
+			<div className='divider-small'></div>
+
+			<div className='comment-list'>
```
Comment (mallionaire) [inline src/components/article-components/ArticleComments.jsx]: You're mapping over a collection of comments here, can you think of an html tag that might be a little more semantic than a div? 
You might want to also have a think about semantic html element you could use for your CommentCard component.

**Review by mallionaire — APPROVED** [review]
Some really nice styling going on here - but please resist using `<div>` where other html tags can offer more clarity to users. 
Have a think about separate loading states, to account for different requests that may complete at different times, more info in comments below.

File: src/components/article-components/ArticleComments.jsx (hunk 2 above)
Comment (Tiny-B) [reply src/components/article-components/ArticleComments.jsx:6]: I didn't realize i was using the same loading state, i must have autopilot it in, of course i'll need a loading state where i actually fetch the comment data

File: src/components/article-components/ArticleComments.jsx (hunk 3 above)
Comment (Tiny-B) [reply src/components/article-components/ArticleComments.jsx]: no excuse, sorry, this should be a list element with list item elements holding the comment card components, not even sure why i blanked that out, thanks

**Pull request merged**